packages/cubejs-schema-compiler/src/compiler/CubeValidator.js

```diff
@@ -633,13 +635,35 @@ export class CubeValidator {
 
     if (result.error != null) {
       errorReporter.error(formatErrorMessage(result.error), result.error);
-    } else {
+    } else if (this.validateName(cube, errorReporter)) {
```

Comment:
Name clash should be done for every cube, not only for cube-view pairs. It should also respect GraphQL name ambiguity.

Reply:
1. I believe IT IS done for every cube now.
2. What is GraphQL name ambiguity?

Reply:
@paveltiunov This is only used to customize the error message for cube-view pairs: https://github.com/cube-js/cube/pull/6610/files#diff-d3163ebccceb844ce7de9dd1aae59f3fe02d4cee03582cc6e6ac90dff0c9231cR650-R652